python/taichi/tools/video.py

```diff
@@ -103,7 +106,15 @@ def __init__(self,
         self.automatic_build = automatic_build
 
     def get_output_filename(self, suffix):
-        return os.path.join(self.directory, 'video' + suffix)
+        if not self.video_filename:
+            return os.path.join(self.directory, 'video' + suffix)
+        else:
```

Comment:
Remove the redundant `else` to make pylint happy :-)

Reply:
Thx for the instruction! Hope pylint is happy now with the follow-up commit!